react/features/conference/functions.native.js

```diff
@@ -15,7 +15,22 @@ export * from './functions.any';
  * @returns {boolean}.
  */
 export function getDisablePolls(stateful: Object) {
-    const state = toState(stateful['features/base/config']);
+    const state = toState(stateful)['features/base/config'];
 
     return state.disablePolls;
 }
+
+/**
+ *
+ * Returns true if a conference room is valid.
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's
+ * {@code getState} function to be used to retrieve the state
+ * features/base/config.
+ * @returns {boolean}.
```

Comment:
This does not return a boolean. I guess you want to `return Boolean(state.room)` ?

Reply:
This helper is no longer needed. Deleted it.